tests/unit/test-firefox/test.firefox.js

```diff
@@ -311,48 +322,84 @@ describe('firefox', () => {
 
     it('configures a profile', () => withBaseProfile(
       (baseProfile) => {
-        const configureThisProfile =
+        const configureProfile =
           sinon.spy((profile) => Promise.resolve(profile));
+        const profileFinder = () => {
```

Comment:
it would be nicer if we can keep the same naming convention in all the tests, e.g. this profileFinder identifier is the function that creates the fake Finder instance in some of the tests and the fake Finder instance in other tests.

Also, if you name these identifier like the params property names, you can use the ES6 short syntax, eg.

```js
const createProfileFinder = () => ...;
const configureThisProfile = () => ...;

...

return firefox.useProfile(profilePath, {app, configureThisProfile, createProfileFinder});
```